<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPLineEditClass</class>
 <widget class="QWidget" name="IPLineEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>36</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget_" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget_{background:transparent; border:1px solid #D2D2D2;}
QWidget#mainWidget_:!enabled{background:#F5F5F5;}
QWidget:!enabled{background:#F5F5F5;}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtIP1_">
        <property name="styleSheet">
         <string notr="true">background:transparent;font:14px Arial;border:none;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDot1_">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px SIMHEI;</string>
        </property>
        <property name="text">
         <string>•</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtIP2_">
        <property name="styleSheet">
         <string notr="true">background:transparent;font:14px Arial;border:none;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDot2_">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px SIMHEI;</string>
        </property>
        <property name="text">
         <string>•</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtIP3_">
        <property name="styleSheet">
         <string notr="true">background:transparent;font:14px Arial;border:none;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDot3_">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:14px SIMHEI;</string>
        </property>
        <property name="text">
         <string>•</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtIP4_">
        <property name="styleSheet">
         <string notr="true">background:transparent;font:14px Arial;border:none;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
